synapse/models/infotech.py

```diff
@@ -1255,6 +1259,45 @@ def getModelDefs(self):
                         'doc': 'An array of ATT&CK technique IDs addressed by the mitigation.',
                     }),
                 )),
+                ('it:mitre:attack:campaign', {}, (
+                    ('name', ('ou:campname', {}), {
+                        'doc': 'The primary name for the ATT&CK campaign.',
+                    }),
+                    ('names', ('array', {'type': 'ou:campname', 'uniq': True, 'sorted': True}), {
+                        'doc': 'An array of alternate names for the ATT&CK campaign.',
+                    }),
+                    ('desc', ('str', {'strip': True}), {
+                        'doc': 'A description of the ATT&CK campaign.',
+                        'disp': {'hint': 'text'},
+                    }),
+                    ('url', ('inet:url', {}), {
```

Comment:
Missing a :period ival (simlar to ou:campaign:period) for MITRE's reported "first seen" and "last seen" duration for the campaign.

Reply:
I was using the `.seen` for that but I can add `:period`.